### ISSUE TITLE
Add auto backup to r2 every hours and keep only 24h

### ISSUE THREAD

**Issue by riderx** [Cap-go/terraform_clickhouse#2]

**Comment by ayewo** [comment]
@riderx is something like this https://github.com/tiredofit/docker-db-backup/tree/main what you had in mind?

Here's an example compose file: https://github.com/tiredofit/docker-db-backup/blob/f7f72ba2c1c7b82aafaed1d8c97b37230fcd074f/examples/compose.yml#L19-https://github.com/tiredofit/docker-db-backup/blob/f7f72ba2c1c7b82aafaed1d8c97b37230fcd074f/examples/compose.yml#L19-L47

**Comment by riderx** [comment]
yes @ayewo if you manage to make it work with r2 as backup location we are good !
